sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2314,6 +2317,64 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -InformationBarriersSuspension
+
+The InformationBarriersSuspension setting enables information barriers in SharePoint and OneDrive when set to $false, or disables information barriers when set to $true.
```

Comment:
Somehow I think it reads better like this

_When InformationBarriersSuspension parameter is set to $false, information barriers in SharePoint and OneDrive is enabled, when set to $true, it is disabled._ 

@JohanFreelancer9 what do you think?

Reply:
Updated.